glui_edittext.cpp

```diff
@@ -1133,34 +1110,25 @@ void   GLUI_EditText::set_int_limits( int low, int high, int limit_type )
 
 /************************************ GLUI_EditText::set_numeric_text() ******/
 
-void    GLUI_EditText::set_numeric_text( void )
+void GLUI_EditText::set_numeric_text( void )
 {
   char buf_num[200];
-  int  i, text_len;
-
-  if ( data_type == GLUI_EDITTEXT_FLOAT ) {
-    sprintf( buf_num, "%#g", float_val );
-
-    num_periods = 0;
-    text_len = (int) strlen(buf_num);
-    for ( i=0; i<text_len; i++ )
-      if ( buf_num[i] == '.' )
-        num_periods++;
-
+  if(data_type==GLUI_EDITTEXT_FLOAT)
+  {
+    snprintf( buf_num, sizeof(buf_num), "%#g", float_val );
```

Comment:
@mick-p1982 Indeed `snprintf` is a better story here than `sprintf`.  (Well spotted!)  But I filed https://github.com/libglui/glui/pull/70 because I think tinyformat is an even better story than `snprintf` nowadays.